parts/k8s/kubernetesconfigs.sh

```diff
@@ -408,6 +408,23 @@ configAddons() {
 }
 
 configGPUDrivers() {
+    # only install the runtime since nvidia-docker2 has a hard dep on docker CE packages.
```

Comment:
does any of this rely on a working outbound connection?

Reply:
the apt stuff does

Reply:
any way to move that stuff in installGPUDrivers? I'm guessing if you didn't already it's probably because we can't but if we leave it here we're basically closing the door on no outbound clusters with GPU for Azure Stack.

Reply:
Let's mark it as a follow-up item, as it'll be be tricky to do that. Basically what we'll have to do is pre-download everything and then manually use dpkg to install local packages.

Reply:
sure, let's open an issue and add it to the backlog